common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -5,35 +5,40 @@
 from django.utils import timezone
 
 from openedx.features.smart_referral.models import SmartReferral
-from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_emails
 
 CONTACT_EMAIL = 'contact_email'
 DAYS_TO_SEND_FOLLOW_UP_EMAIL = 3
 
 
 class Command(BaseCommand):
     help = """
-    Send follow up referral email to all those contacts
+    Send follow-up referral email to all those contacts
     """
 
     def handle(self, *args, **options):
         emails_list = []
-        three_days_old_date = timezone.now().date() - timedelta(days=DAYS_TO_SEND_FOLLOW_UP_EMAIL)
+        referral_threshold_date = timezone.now() - timedelta(days=DAYS_TO_SEND_FOLLOW_UP_EMAIL)
 
-        three_days_old_referral = SmartReferral.objects.filter(
-            created__date__lte=three_days_old_date,
+        three_days_old_referral_emails = SmartReferral.objects.filter(
```

Comment:
we can update the variable name to referral_threshold_date_emails to make it generic so that if the number of days change in future, we only have to change the value in the constant and not the variable name. Also, it will make it consistent with the referral_threshold_date variable.

Reply:
This variable scope is limited to the command. It wouldn't affect our system. So if in future the number of days would be changed we will change the variable without any complication.  It won't have any ripple effects on our system. So I guess 'three_days_old_referral_emails' name is more understandable and explain it's purpose.

Reply:
I also agree with @harris-irfan .. It would be great if we can make this variable name generic here..

Reply:
done